tests/components/tado/util.py

```diff
@@ -165,3 +165,8 @@ async def async_init_integration(
         if not skip_setup:
             await hass.config_entries.async_setup(entry.entry_id)
             await hass.async_block_till_done()
+
+
+def dummy_tado_connector(hass: HomeAssistant, fallback) -> HomeAssistant:
```

Comment:
The return type annotation is incorrect.

Also, I'd suggest to move this function to the test module to make the code easier to read.